nvtabular/tags.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+
+class DefaultTags(Enum):
+    # Feature types
+    CATEGORICAL = "categorical"
+    CONTINUOUS = "continuous"
+    LIST = "list"
+    IMAGE = "image"
+    TEXT = "text"
+    TEXT_TOKENIZED = "text_tokenized"
+    TIME = "time"
+
+    # Feature context
+    USER = "user"
+    ITEM = "item"
+    ITEM_ID = ["item", "item_id"]
```

Comment:
Seems like the type of the tags here should be consistent, so either `str` or `list` based on what's here now. I lean toward making them single strings and handling combinations by setting multiple tags on the schema object. Thoughts, @benfred?